INPTP_AppForFixing/MainForm.cs

```diff
@@ -11,10 +11,11 @@
 namespace INPTP_AppForFixing
 {
     public partial class MainForm : Form
-    {
-
+    {       
+        private HashSet<Boss> employee;
```

Comment:
do we really need two different sets? I would expect that organizational structure is hierarchical and that i will see that structure in gui

Reply:
No we do not need two sets. I forget on hashSet in Boss.